server/container_update_resources_test.go

```diff
@@ -24,19 +25,32 @@ var _ = t.Describe("UpdateContainerResources", func() {
 	t.Describe("UpdateContainerResources", func() {
 		It("should succeed", func() {
 			// Given
-			addContainerAndSandbox()
+			testContainer.SetSpec(&specs.Spec{
+				Linux: &specs.Linux{
+					Resources: &specs.LinuxResources{},
+				},
+			})
 			testContainer.SetState(&oci.ContainerState{
 				State: specs.State{Status: oci.ContainerStateRunning},
 			})
+			addContainerAndSandbox()
 
 			// When
 			response, err := sut.UpdateContainerResources(context.Background(),
 				&pb.UpdateContainerResourcesRequest{
-					ContainerId: testContainer.ID()})
+					ContainerId: testContainer.ID(),
+					Linux: &pb.LinuxContainerResources{
+						CpusetCpus: "1",
```

Comment:
I'd keep the old test - to check that if `CpusetCpus` isn't updated it keeps its zero-value, and I'd add at least another test with a more complex allocation like `0-3,12-15`

Reply:
will do

Reply:
done